resources/puppetlabs/lein-ezbake/template/global/tasks/build.rake

```diff
@@ -163,13 +141,8 @@ namespace :pl do
 
     desc "trigger jenkins packaging job with local auth"
     task :trigger_build_local_auth => "pl:fetch" do
```

Comment:
When I looked at this job I wondered if we even need it at all. In fact, I questioned that wit the entire file because we don't have a Jenkins instance. So is it easier to get rid of it altogether?

Reply:
there are a few rake tasks we don't use, I want to remove them in another PR